README.md

```diff
@@ -53,4 +53,8 @@ In order to execute code submissions using Docker, source code files are written
 - We use a [Prometheus Exporter](https://github.com/discourse/prometheus_exporter) and a [Telegraf Client](https://github.com/jgraichen/telegraf-ruby)
 - The Telegraf client collects the data from the Prometheus endpoint, adds its own datasets and forwards them to an InfluxDB
 - The Prometheus Exporter must be started separately **before** running the Rails server via `bundle exec prometheus_exporter`
-- The InfluxDB data can be visualized using Grafana, for example. There is also an adapted [dashboard](docs/grafana/prometheus_exporter_grafana_dashboard.json) for this purpose
\ No newline at end of file
+- The InfluxDB data can be visualized using Grafana, for example. There is also an adapted [dashboard](docs/grafana/prometheus_exporter_grafana_dashboard.json) for this purpose
+
+
+## Additional Note
+- As the underlying rails server uses port 3000 as the default seeting, you can use [authbind] (https://www.mwells.org/coding/2016/authbind-port-80-443/) to bind it to the regular 80/443 port.
```

Comment:
According to our `puma.rb`, we use 7000 as the default port in development and production mode. It might be confusing to have another "default port" listed here, I think. 

https://github.com/openHPI/codeocean/blob/53a1d32241e0e6d348299a2377511ec9e7d96859/config/puma.rb#L20

Reply:
Good point, I will remove the reference to any default port